netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -14,6 +14,165 @@
 #define EXPIRY_TIME 60000 // 60 seconds in ms.
 #define CONVERT_100NS_UNITS_TO_MS(x) ((x) / 10000)
 
+#define NET_EBPF_EXT_SOCK_ADDR_CLASSIFY_MESSAGE "NetEbpfExtSockAddrClassify"
+
+#define NET_EBPF_EXT_LOG_SOCK_ADDR_CLASSIFY_IPV4(                                                              \
+    trace_level, message, handle, protocol, source_ip, source_port, destination_ip, destination_port, verdict) \
+    TraceLoggingWrite(                                                                                         \
+        net_ebpf_ext_tracelog_provider,                                                                        \
+        NET_EBPF_EXT_SOCK_ADDR_CLASSIFY_MESSAGE,                                                               \
+        TraceLoggingLevel(trace_level),                                                                        \
+        TraceLoggingKeyword(NET_EBPF_EXT_TRACELOG_KEYWORD_SOCK_ADDR),                                          \
+        TraceLoggingString(message, "Message"),                                                                \
+        TraceLoggingUInt64((handle), "TransportEndpointHandle"),                                               \
+        TraceLoggingUInt64((protocol), "Protocol"),                                                            \
+        TraceLoggingIPv4Address((source_ip), "src_ip"),                                                        \
+        TraceLoggingUInt16((source_port), "src_port"),                                                         \
+        TraceLoggingIPv4Address((destination_ip), "dst_ip"),                                                   \
+        TraceLoggingUInt16((destination_port), "dst_port"),                                                    \
```

Comment:
Why are src and dst abbreviated?  The other lines (26-28, 33) are not.  Consistency would be good.

Reply:
I had deliberately abbreviated these strings (there were originally added in my earlier PR) to reduce trace file size. In the cases of always-on tracing, minimizing file size helps.

Reply:
the length of the string in line 27, 43, etc contribute far more than these lines, so if minimizing file size is important, then change those other lines.

Reply:
I have dis-abbreviated them. If you are concerned about the size of the tracelog file then please create an issue @saxena-anurag .